suite-native/app/src/navigation/RootStackNavigator.tsx

```diff
@@ -82,6 +77,15 @@ export const RootStackNavigator = () => {
             />
             <RootStack.Screen name={RootStackRoutes.ReceiveModal} component={ReceiveModalScreen} />
             <RootStack.Screen name={RootStackRoutes.DeviceInfo} component={DeviceInfoModalScreen} />
+            <RootStack.Screen
+                name={RootStackRoutes.ConnectDevice}
+                component={ConnectDeviceStackNavigator}
+                options={{
+                    ...stackNavigationOptionsConfig,
+                    animation: 'slide_from_bottom',
+                    presentation: 'modal',
```

Comment:
This breaks UI a bit, at least on iOS. (see the space on top
![Simulator Screenshot - iPhone 15 - 2024-02-27 at 15 11 01](https://github.com/trezor/trezor-suite/assets/2011829/02b1aa00-3534-498f-b7ca-920e7cb9430f)
![Simulator Screenshot - iPhone 15 - 2024-02-27 at 15 12 27](https://github.com/trezor/trezor-suite/assets/2011829/a8022f91-1106-4d3a-8b3e-fa35f170bb97)
 of modal view)

Reply:
😮 haven't tested on ios (and I believe that if it breaks only on ios, it's not a real issue) but I'll double check

Reply:
0fcc3bb08e9e0e68ce25d3d25dce15aa4ec4426d @vytick could you please check it on ios again? 

I removed `presentation: 'modal'`, 'slide_from_bottom' animation is all we want. We don't want closing by swipe down on iOS for this full screen modal with X button. But I'm not sure if it fixes the layout of this modal.